nixos/tests/netdata.nix

```diff
@@ -25,6 +25,7 @@ import ./make-test-python.nix ({ pkgs, ...} : {
 
     # check if the netdata main page loads.
     netdata.succeed("curl --fail http://localhost:19999/")
+    netdata.succeed("sleep 4")
```

Comment:
Why was this needed? Should this not be covered by `wait_until_succeeds`?

Reply:
without `sleep 4` - error:
```
netdata: waiting for success: curl -s http://localhost:19999/api/v1/data\?chart=users.preads | jq -e '[.data[range(10)][.labels | indices("root")[0]]] | add | . > 0'
netdata # parse error: Invalid numeric literal at line 1, column 6
```

Reply:
The netdata process cannot collect statistics `users.preads`